components/ProductCardNew/index.js

```diff
@@ -0,0 +1,17 @@
+import { BUTTON_PRIMARY } from "@/utils/button";
+import { Button } from "../Button";
+import { StyledCard } from "../StyledCard/index";
+import { ACTION_EDIT } from "@/utils/websiteCard";
```

Comment:
Really minor suggestion, but we could group imports by type for readability. 

```suggestion
import { Button } from "../Button";
import { StyledCard } from "../StyledCard/index";
import { BUTTON_PRIMARY } from "@/utils/button";
import { ACTION_EDIT } from "@/utils/websiteCard";
```